test/integration/targets/ansible-galaxy/runme.sh

```diff
@@ -105,5 +105,40 @@ EOF
 popd # ${galaxy_testdir}
 rm -fr "${galaxy_testdir}"
 
+#################################
+# ansible-galaxy collection tests
+#################################
+
+f_ansible_galaxy_status \
+    "collection init tests to make sure the relative dir logic works"
+galaxy_testdir=$(mktemp -d)
+pushd "${galaxy_testdir}"
+
+    ansible-galaxy collection init ansible_test.my_collection
+
+    # Test that the collection skeleton was created in the expected directory
+    for galaxy_collection_dir in "docs" "plugins" "roles"
+    do
+        [[ -d "${galaxy_testdir}/ansible_test/my_collection/${galaxy_collection_dir}" ]]
+    done
+
+popd # ${galaxy_testdir}
+rm -fr "${galaxy_testdir}"
+
+f_ansible_galaxy_status \
+    "collection init tests to make sure the --init-path logic works"
+galaxy_testdir=$(mktemp -d)
+pushd "${galaxy_testdir}"
+
+    ansible-galaxy collection init ansible_test.my_collection --init-path "${galaxy_testdir}/test"
+
+    # Test that the collection skeleton was created in the expected directory
+    for galaxy_collection_dir in "docs" "plugins" "roles"
+    do
+        [[ -d "${galaxy_testdir}/test/ansible_test/my_collection/${galaxy_collection_dir}" ]]
+    done
+
+popd # ${galaxy_testdir}
+rm -fr "${galaxy_testdir}"
```

Comment:
In general, it's better to implement a `trap` with a cleanup function here.
Example: https://github.com/ansible/ansible/pull/38111/files#diff-aaa2d0f79a3acc45dc595393859b5146R25

Reply:
There's already an existing trap https://github.com/ansible/ansible/blob/devel/test/integration/targets/ansible-galaxy/runme.sh#L7 in the same script. It was my assumption that adding a new trap statement would replace the existing one so I just keep the flow as it was already written.

Reply:
well, you can just extend that trap :)